tests/Temporalio.Tests/Activities/ActivityDefinitionTests.cs

```diff
@@ -1,3 +1,5 @@
+using System.Reflection;
+
 namespace Temporalio.Tests.Activities;
 
 using System.Threading.Tasks;
```

Comment:
```suggestion
namespace Temporalio.Tests.Activities;

using System.Reflection;
using System.Threading.Tasks;
```

Sorry, I know it's pedantic, and we are inconsistent ourselves I've seen across files, but hopefully at least consistent within files

Reply:
Not a problem, totally missed that it was out of order since it was auto added! Updated.